tests/auto/integration/tst_integration.cpp

```
@@ -156,6 +156,22 @@ class tst_integration : public QObject {
     return "finishedInsert not emitted (GPG may have hung or failed to start)";
   }
 
+  // Run a git config command and verify it succeeds.
+  static bool runGitConfig(QProcess &proc, const QString &gitExe,
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)